addPost.php

```diff
@@ -0,0 +1,138 @@
+            <meta http-equiv="X-UA-Compatible" content="IE=edge">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <title>Document</title>
+            <link rel="stylesheet" href="style.css">
+
+        </head>
+        <body>
+            <div>
+                <form action="" method = 'post' enctype="multipart/form-data"> <!-- Comment out if needed -->
+                    <div class = 'flexContainer'>
+
+                        <div class = 'row1Container'>
+                            <div class = 'row1'>
+                                <div class="label">Posting Title</div>
+                                <input type="text" class = 'field' name = 'postingTitle' placeholder = 'ex: MiniFridge' required><br/>
+                            </div>
+
+                            <div class = 'row1'>
+                                <div class="label">Price</div>
+                                $<input type="number" class = 'field' name = 'price' placeholder = '' required><br/>
+                            </div>
+
+                            <div class = 'row1'>
+                                <div class="label">Location</div>
+                                <input type="text" class = 'field' name = 'location' placeholder = 'ex: Umrath Dorm' required><br/>
+                            </div>
+
+                            <div class = 'row1'>
+                                <div class="label">ZipCode</div>
+                                <input type="text" class = 'field' name = 'zipCode' placeholder = '' required><br/>
+                            </div>
+                        </div>
+
+                        <div class = 'row2Container'>
+                            <div class = 'row2'>
+                                <div class="label">Description</div>
+                                <textarea name="description" rows="10" cols="80"></textarea>
+                            </div>
+                        </div>
+
+                        <div class = 'row3Container'>
+                            <fieldset class=''>
+                                <legend>Posting Details</legend>
+                                    <div class = 'row3ContainerInnerFieldSet'>
+                                        <div class = 'row3'>
+                                            <div class = 'column1'>
+                                                <div class="label">Email Address</div>
+                                                <input type="text" class = 'field' name = 'emailInput' placeholder = 'johndoe@gmail.com' required><br/>
+                                                <label for="conditionInput">Condition</label><br/>
+                                                        <select name="conditionInput">
+                                                            <option value="-">-</option> <!-- I'm not too sure how to prevent a user from selecting this-->
```

Comment:
[This article ](https://stackoverflow.com/questions/22033922/how-to-show-disable-html-select-option-in-by-default) looks like this may work, but I would test it: `<option selected="true" disabled="disabled">Choose Tagging</option>`